source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,56 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
```

Comment:
Nitpick: "changes to data" seems like a more appropriate preposition for data and avoids "x of y of z"

Reply:
Fixed